synapse/handlers/pagination.py

```diff
@@ -448,7 +448,7 @@ async def get_messages(
             )
             # We expect `/messages` to use historic pagination tokens by default but
             # `/messages` should still works with live tokens when manually provided.
-            assert from_token.room_key.topological
+            assert from_token.room_key.topological is not None
```

Comment:
Make this more defensive and accurate since `from_token.room_key.topological` is just an `int`.  And `assert 0` will throw an error.

Reply:
@DMRobertson Pretty much. When a room stream token *is* topological, it will be an `int` and `0` is a valid value of `topological_ordering` which we want to allow here ⏩